pyuvdata/uvh5.py

```diff
@@ -642,7 +642,7 @@ def write_uvh5(self, filename, run_check=True, check_extra=True,
             if clobber:
                 print("File exists; clobbering")
             else:
-                raise ValueError("File exists; skipping")
+                print("File exists; skipping")
```

Comment:
Why did you replace this and the next error with a print? I think it is an error if you're trying to write a file and it doesn't get written.

Reply:
I guess I've just never really encountered that in `clobber` statements: and I don't think it was part of `pyuvdata` until recently. It sets up somewhat of a headache when iterating over multiple files, some of which might trigger this which we would want to skip but not fail. Of course we could just catch the exception and pass, but this just seems like a silly barrier, and might let other ValueErrors from an actual code failure slip through.

Said another way, you could reason that this is not at all a code failure: the `clobber` kwarg is set to handle this, so not writing the file is indeed exactly what you want it to do, and __should not__ trigger an exception.